test/utils/timestamp.test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+
+const {format} = require('../../lib/utils/timestamp');
+const timestampOff = require('../../lib/constants/timestamp').none;
+
+describe('timestamp util', () => {
+	it('should be curried', async() => {
+		const date = Date.now();
+		const fmt = 'MMMM DD, YYYY';
+
+		assert.strictEqual(format(fmt, date), format(fmt)(date), 'is curried');
```

Comment:
I think it's irrelevant, an implementation detail that shouldn't affect behavior if changes.